src/components/table/table.tsx

```diff
@@ -683,6 +683,11 @@ export class Table {
         } else {
             row.getElement().classList.remove('active');
         }
+
+        const element = row.getElement().ownerDocument.createElement('div');
+        element.classList.add('interactive-feedback');
+
+        row.getElement().prepend(element);
```

Comment:
the problem with this implementation is that when the table loads, you get one such a div inside each row. 

But every time you click on a row, a new div with this class will be added to that row again
<img width="373" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/24c77c4f-13f2-42b3-8543-bd14f0745ccc">